src/test/java/athena/task/taskfilter/ForecastFilterTest.java

```diff
@@ -75,4 +78,77 @@ void testIsTaskIncluded_day_returnsFalse() {
         boolean isTaskIncluded = forecastFilter.isTaskIncluded(inputTask);
         assertEquals(isTaskIncluded, false);
     }
+
+
+    /**
+     * Check if relevant dates have been removed from task after filtering for a day.
+     */
+    @Test
+    void testremoveExcludedDates_day_returnsTrue() {
```

Comment:
Slight nit: `testRemove` instead of `testremove`.
I also think `day` could be more descriptive, so that from the name we can tell what the test case is testing. How about `filterToday` or something that shows it is testing for filtering based on `TODAY`?

Also the method doesn't return `true` so this is not very precise. How about `returnsOnlyTodayDate`?

 Same goes for the methods below.